site/content/blog/2012-10-22-zap-weekly-releases/index.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "ZAP Weekly Releases"
+description: "I've discussed this with the other ZAP developers, and we've decided to do weekly ZAP releases from the develop branch."
```

Comment:
Some of these summaries look a bit strange in isolation. Do we need them for old blog entries? Maybe just test with ZAPping the Top 10 and then add to new / current content?

Reply:
Sure, that's fine.